### DISCUSSION TITLE
Error when unloading Sugar Beet in Jhonson

### DISCUSSION THREAD

**Discussion by guerrillian** [Courseplay/Courseplay_FS22#2615] [Q&A]
Hi everyone!

As you can see in the screeshot, the Worker (Giants Helper) drive to the wrong unloading spot for sugar beet in Johnson's Farmers Market.

There's a way to make it work as it should be?

If I set the Giants Helper to unload with the Standard work, it works well.

Thanks!

![Screenshot_2](https://github.com/Courseplay/Courseplay_FS22/assets/128705907/2e0a6dd5-b1d7-4bb5-8da2-04b46e06f69e)

**Comment by Tensuko** [comment]
We will have a look at it.
May take a while as we have two other projecs to finish first.

**Reply by guerrillian** [reply]
Thanks for replying!

Have a good day.
